admin/sql/updates/20200921-mbs-10921-unlist-unrelated-edits.sql

```diff
@@ -0,0 +1,282 @@
+        SELECT r.id
+        FROM recording r
+        JOIN artist_credit_name acn ON r.artist_credit = acn.artist_credit
+        WHERE acn.artist = ea.artist
+    )
+    LIMIT 1951 -- of 195080
+)
+RETURNING *;
+
+\echo MBS-10921 (2/12): Unlist “Edit relationship” edits from artists’ history...
+DELETE FROM edit_artist
+WHERE (edit, artist) IN (
+    SELECT ea.edit, ea.artist
+    FROM edit_artist ea
+    JOIN edit e ON ea.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 91
+    AND jsonb_extract_path_text(ed.data, 'type0') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'type1') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int NOT IN (
+        SELECT r.id
+        FROM recording r
+        JOIN artist_credit_name acn ON recording.artist_credit = acn.artist_credit
+        WHERE acn.artist = ea.artist
+    )
+    LIMIT 845 -- of 84449
+)
+RETURNING *;
+
+\echo MBS-10921 (3/12): Unlist “Remove relationship” edits from artists’ history...
+DELETE FROM edit_artist
+WHERE (edit, artist) IN (
+    SELECT ea.edit, ea.artist
+    FROM edit_artist ea
+    JOIN edit e ON ea.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 92
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int NOT IN (
+        SELECT r.id
+        FROM recording r
+        JOIN artist_credit_name acn ON recording.artist_credit = acn.artist_credit
+        WHERE acn.artist = ea.artist
+    )
+    LIMIT 18 -- of 1799
+)
+RETURNING *;
+
+-- II) Recordings
+
+\echo MBS-10921 (4/12): Unlist “Reorder relationships” edits from artists’ history...
+DELETE FROM edit_artist
+WHERE (edit, artist) IN (
+    SELECT ea.edit, ea.artist
+    FROM edit_artist ea
+    JOIN edit e ON ea.edit = e.id
+    JOIN edit_data ed ON e.id = ed.edit
+    WHERE e.id >= 70658947 AND e.id < 71006471
+    AND e.type = 99
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity0', 'id')::int NOT IN (
+        SELECT r.id
+        FROM recording r
+        JOIN artist_credit_name acn ON recording.artist_credit = acn.artist_credit
```

Comment:
```suggestion
        JOIN artist_credit_name acn ON r.artist_credit = acn.artist_credit
```

Reply:
Picked it up with https://github.com/metabrainz/musicbrainz-server/pull/1581/commits/44ccba42ff3e8804c7a8f66f3619aaee393ea2ff.